samples/smart/README.md

```diff
@@ -1,33 +1,60 @@
-# Azure SMART on FHIR ONC (g)(10) Sample
+# Azure ONC (g)(10) & SMART on FHIR Sample
 
-Sample for ONC (g)(10) / SMART on FHIR work. Still under active development.
+This sample demonstrates how Azure Health Data Services can be used to pass the Inferno test for ONC (g)(10) compliance, using Azure Active Directory as the identity provider. While the FHIR Server is the core of this sample, some custom code and routing is required to fully meet the requirements. This sample is therefore *not* using only the FHIR Server but other Azure Services to pass the Inferno tests.
```

Comment:
This sample will support both Azure Health Data Services and Azure API for FHIR, correct ? I think the first line should state Azure Health care APIs instead of Azure Health Data Services.